Gemfile

```diff
@@ -41,6 +41,7 @@ group :development, :test do
   gem 'travis'
   gem 'orderly'
   gem 'capybara'
+  gem 'faker'
```

Comment:
A very useful little gem that makes it easier to make fake data for the seeds (and other things!)